WordPress/Classes/Services/MediaService.m

```diff
@@ -186,6 +187,54 @@ - (void)uploadMedia:(Media *)media
                 failure:failureBlock];
 }
 
+- (void)updateMedia:(Media *)media
+            success:(void (^)())success
+            failure:(void (^)(NSError *error))failure
+{
+    id<MediaServiceRemote> remote = [self remoteForBlog:media.blog];
+    RemoteMedia *remoteMedia = [self remoteMediaFromMedia:media];
+
+    [[ContextManager sharedInstance] saveContext:self.managedObjectContext];
```

Comment:
I'm curious: why do we need to save here? And if we do, shouldn't we wait for the save to finish?